spec/factories/ticket_purchases.rb

```diff
@@ -4,5 +4,11 @@
     conference
     ticket
     quantity 10
+    paid false
+    factory :paid_ticket_purchase do
+    	after(:build) do |ticket_purchase|
```

Comment:
Instead of marking the object as paid and creating physical_tickets by hand, you can use the **pay** method, so maybe you need a payment object too.